### ISSUE TITLE
Let's make a generic "jobs" UI in the table page

### ISSUE THREAD

**Issue by jadami10** [apache/pinot#10470]
There have been several additions to the "jobs" features in Pinot. `forceCommitStatus` and `rebalanceStatus` have both been added recently, but #10088 chose to not include force commits because this is "reload status" only. I'm proposing we change the `Reload Status` button to a `Jobs` button.

This probably require a bit of design to decide what the end state should look like and what are reasonable steps to get there. But first, I want to gauge interest in this work.

**Comment by jadami10** [comment]
cc @Jackie-Jiang @jayeshchoudhary

**Comment by Jackie-Jiang** [comment]
cc @swaminathanmanish

**Comment by swaminathanmanish** [comment]
Yes @jadami10. We need to revisit the UI for this. @Jackie-Jiang and I were talking about this as as well. We may need a drop-down for the list of jobTypes that the user can choose from. 
@jayeshchoudhary - What are you thoughts?

**Comment by jayeshchoudhary** [comment]
I don't have the context about recent changes so not very clear about the requirements
I can start once the requirements are clear.

<img width="1728" alt="Screenshot 2023-03-28 at 3 03 49 PM" src="https://user-images.githubusercontent.com/41536903/228197460-b3ebf967-30d3-4f17-8463-e502b05b2a63.png">


I think we want to change the behavior of `Reload Status` button. 
We recently changed the UI for this button, clicking reload status button now shows two tabs i.e jobs and status 

### Current Behaviour of Reload Status button

https://user-images.githubusercontent.com/41536903/228197216-4026c5bf-f4af-4755-8e8a-4d37dc064bd1.mp4

**Comment by swaminathanmanish** [comment]
Thanks @jayeshchoudhary .Yes you are right about changing reload status to jobs (make it generic). Recently we introduced tableRebalance status as well and now we have 3 types of jobs (reload, force_commit, rebalance). Does it make sense to provide a list/drop-down from the jobs button that lets the user choose the jobType to see status ?

It looks like reload status already has the capability to show different jobTypes (one page to show all jobTypes) ?

**Comment by npawar** [comment]
this is a good enhancement to the job status UI. @swaminathanmanish let's figure out exactly what we'd like to show and then work with @jayeshchoudhary by providing rough wireframes and relevant APIs. Will DM you
